### PR TITLE
[CodePush] and constants and configuration

### PR DESCRIPTION
**Pull request by sergey-akhalkov**
Please review and feel free to edit if you see any typos or grammar issues.

### CODE REVIEW

File: sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/CodePushConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+     * See {@link com.microsoft.appcenter.codepush.enums.CodePushUpdateState} for details.
+     */
+    private String packageHash;
+
+    /**
+     * Get the appVersion value.
+     *
+     * @return appVersion value
+     */
+    public String getAppVersion(){
```
Comment (MatkovIvan) [inline sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/CodePushConfiguration.java]: Missing space

File: sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/CodePushConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        return this.appVersion;
+    }
+
+    /**
+     * Get the clientUniqueId value
```
Comment (MatkovIvan) [inline sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/CodePushConfiguration.java]: Missing dot, recheck all in this file

File: sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/CodePushConfiguration.java (hunk 1 above)
Comment (sergey-akhalkov) [reply sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/CodePushConfiguration.java]: fixed, thanks

File: sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/CodePushConfiguration.java (hunk 2 above)
Comment (sergey-akhalkov) [reply sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/CodePushConfiguration.java]: fixed, thanks!

File: sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/CodePushConfiguration.java [hunk 3]
```diff
@@ -80,49 +83,75 @@ public String getPackageHash() {
...
+     * @return this instance.
      */
     @SuppressWarnings("WeakerAccess")
-    public void setAppVersion(String appVersion) {
-        this.appVersion = appVersion;
+    public CodePushConfiguration setAppVersion(String appVersion) {
+        if (appVersion != null) {
+            this.appVersion = appVersion;
+        } else {
+            AppCenterLog.error(AppCenter.LOG_TAG, "\"appVersion\" property cannot be null.");
```
Comment (MatkovIvan) [inline sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/CodePushConfiguration.java]: Use log tag from module

File: sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/CodePushConstants.java [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+    /**
+     * Key for getting/storing previous CodePush update.
+     */
+    public static final String PREVIOUS_PACKAGE_KEY = "previousPackage";
+
+    /**
+     * Key for specifying app entry point (JS bundle in React Native, <code>index.js</code> for Cordova).
+     */
+    public static final String APP_ENTRY_POINT_PATH_KEY = "appEntryPoint";
```
Comment (annakocheshkova) [inline sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/CodePushConstants.java]: RN/Cordova specific?

File: sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/CodePushConfiguration.java (hunk 3 above)
Comment (sergey-akhalkov) [reply sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/CodePushConfiguration.java]: Fixed, thanks

File: sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/CodePushConstants.java (hunk 4 above)
Comment (sergey-akhalkov) [reply sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/CodePushConstants.java]: Added clarification, please verify and let me know if you still have a concern regarding this configuration.

**Pull request closed without merging**